spanner/cloud-client/snippets.py

```diff
@@ -1397,6 +1449,34 @@ def query_data_with_string(instance_id, database_id):
     # [END spanner_query_with_string_parameter]
 
 
+def query_data_with_numeric_parameter(instance_id, database_id):
+    """Queries sample data using SQL with a NUMERIC parameter. """
+    # [START spanner_query_with_numeric_parameter]
+    # instance_id = "your-spanner-instance"
```

Comment:
nits: should we add these comments for the above samples?

Reply:
I was matching similar samples. The difference seems to be that the region here begins inside the the function so the comments show that they are defined, whereas the other sample region tags after around the function which means the variables are defined as arguments.

We can consider making the region tags more consistent but that's out of scope for this PR